From 3d8a425ef426019c49abea20baa58461900c731d Mon Sep 17 00:00:00 2001
From: "Friedrich W. H. Kossebau" <kossebau@kde.org>
Date: Thu, 29 Oct 2020 13:15:22 +0100
Subject: [PATCH] Remove unused explicit dep on and linkage of Solid

(still pulled in via public dep of KIOFileWidgets, but not used by us)
---
 CMakeLists.txt     | 1 -
 src/CMakeLists.txt | 1 -
 2 files changed, 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 72ec982d..bcd101e1 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -30,7 +30,6 @@ find_package(KF5 ${KF5_MIN_VERSION} REQUIRED
     NotifyConfig
     KIO
     Parts
-    Solid
     Wallet
     WidgetsAddons
     GlobalAccel
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 460325f1..680053e2 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -244,7 +244,6 @@ target_link_libraries(konversation
     KF5::KIOFileWidgets
     KF5::KIOWidgets
     KF5::Parts
-    KF5::Solid
     KF5::Wallet
     KF5::WidgetsAddons
     KF5::GlobalAccel
-- 
GitLab

